src/components/ClusterNodes/ClusterDetails.tsx

```diff
@@ -537,12 +537,30 @@ export default function ClusterDetails({ imageList, isSuperAdmin, namespaceList,
         )
     }
 
+    const renderSortDirection = (column: ColumnMetadataType) : JSX.Element => {
+        if(column.isSortingAllowed) {
+            if(sortByColumn.value === column.value) {
+                return (
+                    <>
+                        <span className={`sort-icon ${sortOrder == OrderBy.DESC ? 'desc' : '' } ml-4`}></span>
+                    </>
+                )
+            } else {
+                return (
+                    <>
```

Comment:
remove diamond brac
& use <span   .....  /> if use not passing anything

Reply:
done